src/main/scala/io/qbeast/core/model/MetadataManager.scala

```diff
@@ -62,7 +62,8 @@ trait MetadataManager[DataSchema, FileDescriptor, QbeastOptions] {
       tableID: QTableID,
       schema: DataSchema,
       options: QbeastOptions,
-      append: Boolean)(writer: => (TableChanges, IISeq[FileDescriptor])): Unit
+      append: Boolean)(
+      writer: => (TableChanges, IISeq[FileDescriptor], IISeq[FileDescriptor])): Unit
```

Comment:
Why are we returning 2 lists of file descriptors? Can you update the method docs to describe it?

Reply:
Fixed